datasources/includes/WA.ejs

```diff
@@ -1,7 +1,17 @@
     {
       "type": "ckan",
       "name": "Western Australian Government",
-      "url": "http://catalogue.beta.data.wa.gov.au/",
+      "url": "https://catalogue.data.wa.gov.au/",
       "groupBy": "organization",
-      "filterQuery": [ "q=data_homepage:*_Public_Services*&fq=res_format:WMS" ]
+      "filterQuery": [ 'q=access_level:open&fq=res_format:(GeoJSON OR "OGC WMS" OR "OGC WFS" OR WFS OR "API ArcGIS Server Feature Service" OR "API ArcGIS Server Map Service" OR "API ArcGIS Map Service" OR "Esri REST")' ],
```

Comment:
The query will return results with `access_level:open` at top level. However, there are mixed `access_level:open` and `access_level:open_login` in the `resources` list. The latter only grants access to authenticated users which is not supported by the `National Map`. 
Besides, resource list includes many formats like CSV, SHP, PDF, etc.